src/applications/pact-act/sass/pact-act.scss

```diff
@@ -18,4 +20,28 @@
       margin-left: 8px;
     }
   }
+
+  .pact-act-form-question-header {
+    font-family:  "Bitter", "Georgia", "Cambria", "Times New Roman", "Times", serif;
+    font-size: 2em;
+    line-height: 1.5;
+    margin-bottom: 0px;
+    max-width: 46rem;
+  }
+
+  .pact-act-form-question-error {
+    border-left: 0.4rem solid #cd2026;
```

Comment:
Remember to swap these hexes out for the proper color if we can find it in the design system